cmd/auctionminion/main.go

```diff
@@ -61,7 +61,7 @@ var stateFile = flag.String("statefile", "auctionminion.state", "State/config fi
 var initFlag = flag.Bool("init", false, "Initialize state file")
 
 type minionConfig struct {
-	AuctionKey DigestChecksummed
+	AuctionKey crypto.Digest
```

Comment:
Should be `basics.Address` rather than `crypto.Digest` if you want it to include the checksum.

Reply:
Agreed. Will fix tonight.

Reply:
Looks like this change broke a bunch of stuff in the auction area - I'm going to try to enlist the help of the relevant subject matter expert (Evan) for some "unsupervised fun" over the break to get it worked out. If that ends up taking a while would you consider taking this PR sans changes in the auction space?

Reply:
Yes - feel free to submit an update without the auction changes and we'll re-evaluate it.